features/facebookImport/src/model/analyses/utils/on-off-facebook-data-restructuring.js

```diff
@@ -116,3 +116,11 @@ export function buildDisplayData(
 
     return getIntoChartStructure(displayData);
 }
+
+export function top5OffFacebookCompanies(facebookAccount) {
+    return [...facebookAccount.offFacebookCompanies]
+        .map((e) => {
+            return { title: e.name, count: e.events.length };
+        })
+        .sort((a, b) => b.count - a.count);
```

Comment:
Not capping at 50?
Also maybe 50 should be in an argument, with a default value.

Reply:
Haha, thats actually funny. It's supposed to be 5 and Off :D with seems like 50. In the end I shifted the logic into the component to only use the first 5 as we need to have the possibility to show all in the future.

Reply:
Naming nit then?

Reply:
Done already :D